### ISSUE TITLE
When installing: The artifact repository requires PHP's zip extension

### ISSUE THREAD

**Issue by xyloweb2** [composer/composer#3921]
I am trying to get the Google Identity Toolket working here: https://developers.google.com/identity-toolkit/quickstart/php
As part of that, I am trying to install Composer by typing: php composer.phar install
I get a message: The artifact repository requires PHP's zip extension
I have PHP 5.5.19, and "ZIP-based phar archives" are enabled, which seems like the closest thing to what it is asking for.
So, I don't understand what might be wrong with my server, or what is wrong with Composer.

**Comment by alcohol** [comment]
composer.phar is the composer application. You don't have to install it, you just run it.

The toolkit you mentioned gives you a zip to download. This zip archive probably contains a composer.json which defines another archive as a source medium. To be able to access archives, you need to enable the [ZIP](http://php.net/manual/en/book.zip.php) extension (completely **not** related to [PHAR](http://php.net/manual/en/book.phar.php)).

**Comment by xyloweb2** [comment]
OK, thanks, I found the right Zip extension and enabled it.  I now get past that error, but get an error "git was not found, check that it is installed and in your PATH env".  I can tell that I have git, but that it is not in the Apache path.  I see that mod_env would normally be the place to fix this (I think), but the documentation specifically says this doesn't work on Apache 2.2, which is what I have.  If someone would tell me how to (at least temporarily) add a directory to my Apache path, I would appreciate it.
(I will close this zip issue here.)

**Comment by alcohol** [comment]
It doesn't need to be in the Apache path? 
Apache is not related to the install steps. 

Composer will use git from the command line when trying to install remote git dependencies. Run `which git` from the command line to find out where git is installed, e.g. on my Mac:

```
$ which git
/usr/local/bin/git
```

Then make sure that path is in your PATH:
```
$ echo $PATH
/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin
```

If it is not, add it to your PATH:
```
$ export PATH=/usr/local/bin:$PATH
```

**Comment by xyloweb2** [comment]
Thanks, this is exactly what I needed (the 'export' command).
FYI: "which git" only returns a result after git is in your path.
I don't know how to figure out where git is if you don't already know.
Luckily, I had already figured out where it was stored, and "export" worked as expected.
One change was that I put the new path -after- the existing path, because I did not want to change how existing things were being located.  So I used something like: export PATH=$PATH:/usr/local/bin
